### DISCUSSION TITLE
Is it possible to set the default map features (e.g. to not show polylines) when changing date ranges?

### DISCUSSION THREAD

**Discussion by micvis** [Freika/dawarich#348] [Ideas]
Firstly, I just want to say this is a great app, hopefully this could be my pathway to not leaving all of my location history on Google!

I have just started importing my Google location history, and when I change the time range on the map, the layer options I've selected revert with "Polylines" and "Heatmap" selected, even when I've changed them to something else. The map type (e.g. OpenStreetMap.HOT) stays with what I've selected though.  

Is there a way to change this behaviour? I have a lot of Google data and the map rendering struggles and at times fails when trying to draw a lot of lines.
 
![image](https://github.com/user-attachments/assets/8f91ce0d-cf47-402a-95b9-7478acd36df5)

**Comment by Freika** [comment]
For now, it's not being saved, but I'll add functionality for map to remember enabled and disabled layers 👍

**Reply by micvis** [reply]
That's great, thanks!

**Comment by Freika** [comment]
Done in 0.34.0
